pkg/controller/utils/capability.go

```diff
@@ -306,8 +308,25 @@ func getOpenAPISchema(capability types.Capability) ([]byte, error) {
 }
 
 // generateOpenAPISchemaFromCapabilityParameter returns the parameter of a definition in cue.Value format
-func generateOpenAPISchemaFromCapabilityParameter(capability types.Capability) ([]byte, error) {
-	return GenerateOpenAPISchemaFromDefinition(capability.Name, capability.CueTemplate)
```

Comment:
GenerateOpenAPISchemaFromDefinition does this func still being used?

Reply:
i will fix it